pilot/pkg/config/kube/ingress/status.go

```diff
@@ -119,7 +119,7 @@ func (s *StatusSyncer) runUpdateStatus(stop <-chan struct{}) {
 }
 
 // updateStatus updates ingress status with the list of IP
-func (s *StatusSyncer) updateStatus(status []corev1.LoadBalancerIngress) error {
+func (s *StatusSyncer) updateStatus(status []v1beta1.IngressLoadBalancerIngress) error {
```

Comment:
why downgrade ingress api

Reply:
Its not downgrade, they moved `core/v1` to the ingress API. And this folder is for v1beta1 ingress. The v1 ingress uses v1/